version-migration/v0.7-to-v0.8-migration/migrated-examples/build.gradle

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *       http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+plugins {
+    id 'java-library'
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    testImplementation("org.junit.jupiter:junit-jupiter-params:5.8.2")
+
+    testImplementation("org.partiql:partiql-lang-kotlin:0.8.0")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
```

Comment:
For the v0.8.0 import, could you try excluding this section? From https://github.com/partiql/partiql-lang-kotlin/issues/858, the additional, `sourceCompatibility` and `targetCompatibility` sections weren't needed for v0.8.0 and only for v0.7.0. I think the build should work without this section needed